openvdb_houdini/openvdb_houdini/SOP_OpenVDB_Activate.cc

```diff
@@ -493,14 +493,20 @@ template <typename GridType>
 static void
 sopDilateVoxels(GridType& grid, int count)
 {
-    openvdb::tools::dilateVoxels(grid.tree(), count);
+    openvdb::tools::dilateActiveLeafValues(grid.tree(), count);
 }
 
 template <typename GridType>
 static void
 sopErodeVoxels(GridType& grid, int count)
 {
-    openvdb::tools::erodeVoxels(grid.tree(), count);
+    openvdb::tools::erodeActiveLeafValues(grid.tree(), count);
+    if (grid.getGridClass() == openvdb::GRID_LEVEL_SET) {
+        openvdb::tools::pruneLevelSet(grid.tree());
+    }
+    else {
+        openvdb::tools::pruneInactive(grid.tree());
+    }
```

Comment:
@jmlait just an FYI, I have not changed the behaviour of the activate SOP in this PR; it still only works on voxels